### PR TITLE
i18n(es): Update `content-collections`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `content-collections` based on this commit 1a2215d3588a7d5884ac2451b18c896c504f33d9

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/content-collections.mdx [hunk 1]
```diff
@@ -564,3 +564,36 @@ const { remarkPluginFrontmatter } = await blogPost.render();
...
+El formato de fecha se especificará en UTC si no se proporciona una zona horaria. Si necesitas especificar una zona horaria, puedes usar el formato [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+
+```markdown title="src/pages/posts/example-post.md"
+---
+title: Mi Publicación de Blog
+pubDate: 2021-07-08T12:00:00-04:00
+---
+```
+
+To render only the `YYYY-MM-DD` from the full UTC timestamp, use the JavaScript `slice` method to remove the timestamp:
```
Comment (Waxer59) [inline src/content/docs/es/guides/content-collections.mdx:589]: Tienes que eliminar esta linea

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🚀

**Pull request merged**